components/buildList/js/buildList.controller.js

```diff
@@ -237,4 +237,242 @@ buildApp.controller('buildListController', ['$scope', '$q', '$interval', 'buildL
 	}
 
 	init();
+}]).controller('buildListController', ['$scope', '$q', '$interval', 'buildListService', function ($scope, $q, $interval, buildListService) {
+	
+	var deferred = $q.defer(),
+		updateBuildsInterval,
+		intervalTime = 5000,
+		timeElapsed = 0;
+
+	/**
+	 * Initializes the functionality of this controller
+	 *  - requests the data
+	 *  - starts to calculate all the processes
+	 * @method init
+	 */
+	function init() {
+		getData();
+		$q.when(deferred.promise, function() {
+			updateBuildsInterval = $interval(updateBuildsList, intervalTime);
+		});
+	}
+
+	/**
+	 * Requests for the build list data
+	 * @method getData
+	 */
+	function getData() {
+		return buildListService.getData().then(function(response) { 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ATrailingWhitespace)